_sass/fabric.scss

```diff
@@ -1,3 +1,8 @@
+@font-face {
+	font-family: "Comic Relief";
+	src: url("/assets/fonts/ComicRelief-Regular.woff2");
```

Comment:
Where did this font come from, what is the license? It would be useful to document this somewhere.

Reply:
It's from here https://github.com/loudifier/Comic-Relief, it's OFL. Maybe a license file beside the font?

Reply:
Yes, reading the license I think it would make sense to put it alongside it.